docs/fitting_types.rst

```diff
@@ -385,6 +385,29 @@ are added squared, ready to be minimized. Unlike in the above example, the
 
   Do not cite the overall :math:`R^2` given by :mod:`symfit`.
 
+Fitting moredimensional datasets
```

Comment:
The mathematical term is multidimensional instead of moredimensional

Reply:
I changed this with the last push